components/BottomDrawerBuilding.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import BottomDrawer from 'rn-bottom-drawer';
+import Building from '../classes/building';
+import Colors from '../constants/Colors';
+
+const styles = StyleSheet.create({
+  contentContainer: {
+    flex: 1,
+    borderBottomWidth: 1,
+    borderTopWidth: 1,
+    borderLeftWidth: 1,
+    borderRightWidth: 1,
+    borderTopLeftRadius: 12,
+    borderTopRightRadius: 12,
+    borderColor: Colors.mapMarkerColor,
+  },
+  titleBuilding: {
+    fontSize: 30,
+    color: Colors.primaryColor,
+    textAlign: 'left',
+  },
+  location: {
+    color: Colors.grey,
+    textAlign: 'left',
+    fontStyle: 'italic',
+  },
+  description: {
+    fontSize: 17,
+    color: Colors.black,
+    textAlign: 'left',
+  },
+});
+
+type displayInfoProps = {
+  building: Building;
+  displayInfo: boolean;
+};
+
+type displayInfoState = {
+  building: Building;
+  displayInfo: boolean;
+};
+
+const TAB_BAR_HEIGHT = 0;
+
+class BottomDrawerBuilding extends Component<displayInfoProps, displayInfoState> {
+  constructor(props) {
+    super(props);
+    const { building, displayInfo } = this.props;
+    this.state = {
+      building,
+      displayInfo,
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    const { displayInfo } = this.props;
+    if (displayInfo !== prevProps.displayInfo) {
+      this.changeState();
+    }
+  }
+
+  changeState() {
+    const { displayInfo } = this.props;
+    const { building } = this.props;
+    this.setState({ displayInfo });
+    this.setState({ building });
+  }
+
+  renderContent = (building: Building) => {
+    return (
+      <View style={styles.contentContainer}>
+        <Text style={styles.titleBuilding}>{building.getDescription()}</Text>
+        <Text style={styles.location}>
+          Location: [{building.getLocation().getLatitude()}, {building.getLocation().getLongitude()}
+          ]
+        </Text>
+        <Text style={styles.description}>
+          insert description here: lorem sepsum blabalbaaddshfsdfsdfs sg dfg dfg hfgf ghgh dfgsdg 56
```

Comment:
why isn't the description from the building added here? I know we wont have good values, but we should utilize the variables still and not have it hard coded

Reply:
I used that because the building description is already used. It doesnt really matter. I wanted to show that the description of the building should go there. I did not show the building description because there are none for now. But definitely, it will replace the hardcoded value